conf.yaml

```diff
@@ -88,6 +88,10 @@ contents:
                 repo:   logstash
                 path:   docs/
                 exclude_branches:   [ 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0 ]
+              -
+                repo:   observability-docs
+                path:   docs/en/observability
+                exclude_branches:   [ 7.9 ]
```

Comment:
If I understand this correctly, this should be a list of the branches of the Installation and Upgrade guide that shouldn't use this as a source. @nik9000, can you confirm that?

Reply:
That's how I understand it as well. It only includes `7.9` because the obs-docs guide is only built in `master`, `7.x`, `7.10`, and `7.9`

Reply:
I see what you're saying. You're right, it should be based on IU guide, not Obs. I'll fix!

Reply:
```suggestion
                exclude_branches:   [ 7.9, 7.8, 7.7, 7.6, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0 ]
```